MINOR: Clarify offsets etc are unsigned integers (#475)
apache/parquet-format: VariantEncoding.md

metadata  |        header         |
          +-----------------------+
          |                       |
          :    dictionary_size    :  <-- unsigned little-endian, `offset_size` bytes
          |                       |
          +-----------------------+
          |                       |

>           :        offset         :  <--  unsigned little-endian, `offset_size` bytes

Reviewer: NIT:
```suggestion
          :        offset         :  <-- unsigned little-endian, `offset_size` bytes
```

          |                       |
          +-----------------------+
                      :
          +-----------------------+
          |                       |

>           :        offset         :  <--  unsigned little-endian, `offset_size` bytes

Reviewer: NIT:
```suggestion
          :        offset         :  <-- unsigned little-endian, `offset_size` bytes
```

          +-----------------------+
```

> The metadata is encoded first with the `header` byte, then `dictionary_size` which is a unsigned little-endian value of `offset_size` bytes, and represents the number of string values in the dictionary.

Reviewer: ```suggestion
The metadata is encoded first with the `header` byte, then `dictionary_size` which is an unsigned little-endian value of `offset_size` bytes, and represents the number of string values in the dictionary.
```

Next, is an `offset` list, which contains `dictionary_size + 1` values.

> Each `offset` is a usigned little-endian value of `offset_size` bytes, and represents the starting byte offset of the i-th string in `bytes`.

Reviewer: ```suggestion
Each `offset` is an unsigned little-endian value of `offset_size` bytes, and represents the starting byte offset of the i-th string in `bytes`.
```

                   |                       |
                   +-----------------------+
```
An object `value_data` begins with `num_elements`, a 1-byte or 4-byte unsigned little-endian value, representing the number of elements in the object.
The size in bytes of `num_elements` is indicated by `is_large` in the `value_header`.
Next, is a list of `field_id` values.

> There are `num_elements` number of entries and each `field_id` is a unsigned little-endian value of `field_id_size` bytes.

Reviewer: ```suggestion
There are `num_elements` number of entries and each `field_id` is an unsigned little-endian value of `field_id_size` bytes.
```

A `field_id` is an index into the dictionary in the metadata.
The `field_id` list is followed by a `field_offset` list.

> There are `num_elements + 1` number of entries and each `field_offset` is a unsigned little-endian value of `field_offset_size` bytes.

Reviewer: ```suggestion
There are `num_elements + 1` number of entries and each `field_offset` is an unsigned little-endian value of `field_offset_size` bytes.
```

                  |                       |
                  +-----------------------+
```
An array `value_data` begins with `num_elements`, a 1-byte or 4-byte unsigned little-endian value, representing the number of elements in the array.
The size in bytes of `num_elements` is indicated by `is_large` in the `value_header`.
Next, is a `field_offset` list.

> There are `num_elements + 1` number of entries and each `field_offset` is a unsigned little-endian value of `field_offset_size` bytes.

Reviewer: ```suggestion
There are `num_elements + 1` number of entries and each `field_offset` is an unsigned little-endian value of `field_offset_size` bytes.
```
